src/lib/drupal/drupalClient.ts

```diff
@@ -20,7 +20,8 @@ export const drupalClient = new DrupalClient(baseUrl, {
     process.env.USE_REDIS === 'true'
       ? redisCache(createRedisClient(process.env.REDIS_URL))
       : null,
-  previewSecret: process.env.DRUPAL_PREVIEW_SECRET,
+  // I think this is deprecated in next-drupal v2.0.0
+  // previewSecret: process.env.DRUPAL_PREVIEW_SECRET,
```

Comment:
@timcosgrove Any idea how this was used before? From what I could tell, `next-drupal` [still uses `DRUPAL_PREVIEW_SECRET`](https://github.com/search?q=repo%3Achapter-three%2Fnext-drupal%20DRUPAL_PREVIEW_SECRET&type=code), but I'm not really sure how to test it out.

Reply:
I think we need to deploy this next branch on a Drupal tugboat to test the preview. I can test that.